portfolio/src/main/webapp/script.js

```diff
@@ -43,4 +43,27 @@ function addRandomQuote() {
     });
 }
 
+/** Adds an HTML element from Servlet. */
+function showComments() {
+  const commentContainer = document.getElementById('comments');
+  commentContainer.innerHTML = '';
+
+  fetch('/data')
+    .then(response => response.json())
+    .then(comments => {
+      comments.forEach(comment => {
+      commentContainer.appendChild(createListElement(comment));
+      });
+    })
+    .catch(error => void console.error(error))
```

Comment:
Missing semi-colon at the end of the line.

Reply:
Changes made.